jenkins/test_edxapp.sh

```diff
@@ -24,12 +24,17 @@ set -x
 git clean -xfd
 
 # Create the virtualenv and install requirements
-mkdir -p venv
-virtualenv venv
-. venv/bin/activate
+#mkdir -p venv
```

Comment:
I understand why you would not want to run these each time, but we'll need them somewhere so we can automate the test runs (even if it's push-button).